app/frontend/src/components/NotificationBar/_NotificationBar_Close.scss

```diff
@@ -0,0 +1,97 @@
+// External Imports
+@use "sass:map";
+
+// Internal Imports
+@use "../Basics" as *;
+
+/*****************************
+*** Notification Bar class ***
+*****************************/
+$themes: "primary", "success", "warning", "danger";
+
+p {
+  padding: 0;
+  margin: 0;
```

Comment:
Let's avoid redefining whole tags for the entire document. This will get very messy down the line.